VERSION

```diff
@@ -1 +1 @@
-1.11.1
+1.11.2
```

Comment:
Two birds in one stone?

I can make a separate PR for that 1 char change if you prefer though.